index.js

```diff
@@ -1,100 +1,66 @@
 'use strict';
 
-/*
- * Dependencies.
- */
-
-var sanitize = require('hast-util-sanitize');
+/* Dependencies. */
+var has = require('has');
 var toH = require('hast-to-hyperscript');
 
-try {
-    var globalCreateElement = require('react').createElement;
-} catch (e) { }
+/* Expose `rehype-react`. */
+module.exports = rehype2react;
 
-var own = {}.hasOwnProperty;
+/* Get React’s `createElement`. */
+var globalCreateElement;
 
-var TABLE_ELEMENTS = ['table', 'thead', 'tbody', 'tfoot', 'tr'];
+try {
+  /* eslint-disable import/no-extraneous-dependencies */
+  globalCreateElement = require('react').createElement;
+} catch (err) {}
```

Comment:
I wonder this code is hacky.  Shall we make `options.createElement` non optional and use it?